ja/messages.json

```diff
@@ -194,11 +194,17 @@
     "Add_GM_functions_to_this_or_window": {
         "message": "this または window に GM 関数を追加"
     },
+    "Add_Selection_Above": {
+        "message": "Add Selection Above"
```

Comment:
Shouldn't we consider removing untranslated messages for now?

Reply:
Either present or absent, If this sentence is used, the English text will be displayed. If delete this, the line numbers are different from ```en/messages.json```.

Reply:
I understand the benefits of aligning line numbers.
However, there is a concern that the developer won't be able to distinguish between intentionally untranslated messages (e.g., "Google Drive") and messages that haven't been translated due to a lack of context.

Reply:
No one can distinguish between them. It can distinguish, if you memorized the meaning. Or comment tags need to be included.

* Strings in the Japanese file that are still in English are either: untranslated, or intentionally untranslated
* Strings that are not in the Japanese file are either: untranslated, or intentionally untranslated

It is also common that developers to add all untranslated strings for all languages.

Reply:
If we edit ja.json while comparing it with en.json using diff tools like WinMerge, I don't necessarily think it's essential to align line numbers.
What do you think?

Reply:
The same line number makes it easy to match the same line in each file. 

Because it doesn't mean more than that. So I will remove it.